src/index.html

```diff
@@ -15,8 +15,86 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner">
+      <a
+        class="img1"
+        href="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1712534400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=oTn3~L--mDnRZKcoPxL8QydsYHXYco4Bp8Rgm3qnrxJZul2VRiIySVkE-22IvcmcudEcgEsHNLsumGF5NzOa1lMBtAM93RiLNbEJeDlxnMRmF03MOXKmzBMbjOY2MeO13BtDRlmFkfsa8sgGcfZY~znCgT0kPtoikEXAlTqXBW2rfUXrR9wmSdYCxkfqbSNt6YOd1KGvgcA0HpAevZkzZCtJojMr3tYZBgJtAaAlWNbmBrkwdOlEEacUBFdQLY1F7DT31l6ja8mwCaZEfGgiwDsoHFRQmzDY6I1XCZYJHfY93kZH08u-J~PXvuTSG4rb7Q2nZsle7IDz2QYt7tXYrQ__"
+      >
+        <img
+          class="img1"
+          href="moyo_1.png"
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1712534400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=oTn3~L--mDnRZKcoPxL8QydsYHXYco4Bp8Rgm3qnrxJZul2VRiIySVkE-22IvcmcudEcgEsHNLsumGF5NzOa1lMBtAM93RiLNbEJeDlxnMRmF03MOXKmzBMbjOY2MeO13BtDRlmFkfsa8sgGcfZY~znCgT0kPtoikEXAlTqXBW2rfUXrR9wmSdYCxkfqbSNt6YOd1KGvgcA0HpAevZkzZCtJojMr3tYZBgJtAaAlWNbmBrkwdOlEEacUBFdQLY1F7DT31l6ja8mwCaZEfGgiwDsoHFRQmzDY6I1XCZYJHfY93kZH08u-J~PXvuTSG4rb7Q2nZsle7IDz2QYt7tXYrQ__"
+          alt="MOYO.img"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="#APPLE"
+          class="box box-1 is-active"
+        >
+          APPLE
+        </a>
+        <a
+          href="#SAMSUNG"
+          class="box box-2"
+        >
+          SAMSUNG
+        </a>
+        <a
+          href="#SMARTPHONES"
+          class="box box-3"
```

Comment:
<img width="672" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/53a6d345-ec03-40c7-aab4-02d3a6adc98d">

Reply:
and fix your html structure in the nav tag like in the example above